feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -64,12 +65,16 @@ func (srv *CentralSessionController) CreateSession(
 	request *protos.CreateSessionRequest,
 ) (*protos.CreateSessionResponse, error) {
 	if err := validateCreateSessionRequest(request); err != nil {
+		err = fmt.Errorf("failed to validate CreateSessionRequest: %s", err)
+		glog.Error(err)
```

Comment:
Sorry 😓I  forgot to mention that since we are reducing the amount of logs, lets add verbosity 

```
err = fmt.Errorf("CreateSession failed to validate request: %s", err)
```

This way you know this error comes from createsession
You should do the same for the others

Reply:
Done